src/cmake/Helpers.cmake

```diff
@@ -176,11 +176,6 @@ function(alicevision_add_software software_name)
     PROPERTY FOLDER ${SOFTWARE_FOLDER}
   )
 
-  set_target_properties(${software_name}
-     PROPERTIES SOVERSION ${ALICEVISION_SOFTWARE_VERSION_MAJOR}
-     VERSION "${ALICEVISION_SOFTWARE_VERSION_MAJOR}.${ALICEVISION_SOFTWARE_VERSION_MINOR}"
-  )
-
```

Comment:
On linux, it generates softwares with the version at the end: `software-MAJOR.MINOR` and the corresponding symlink without the versioning.